src/world/Actor/Player.h

```diff
@@ -1114,6 +1117,14 @@ namespace Sapphire::Entity
 
     Common::PlayerTeleportQuery m_teleportQuery;
 
+    struct PlayerDyeingInfo
+    {
+      uint32_t itemToDyeContainer;
+      uint32_t itemToDyeSlot;
+      uint32_t dyeBagContainer;
+      uint32_t dyeBagSlot;
+    }; PlayerDyeingInfo m_dyeingInfo;
```

Comment:
I would be inclined to put this struct elsewhere, though I'm not entirely sure on where this should go either. either way, the actual field should be on it's own line, not right after the semicolon